recipe/meta.yaml

```diff
@@ -22,12 +22,11 @@ requirements:
     - wheel
     - setuptools >=30.0.0
   run:
-    - python >=3.6
-    # mccabe 0.7.0 is a noarch python package now and it hasn't broken changes for flake8
-    - mccabe >=0.6.0,<0.8.0
+    - python
+    - mccabe >=0.6.0,<0.7.0
     - pycodestyle >=2.8.0,<2.9.0
     - pyflakes >=2.4.0,<2.5.0
-    - importlib-metadata
+    - importlib-metadata <4.3
```

Comment:
This seems to be only needed for versions less then 3.8. https://gitlab.com/pycqa/flake8/-/blob/master/setup.cfg#L44

Reply:
Because we only rebuilding v4.0.1 we won't change the architecture. It will be noarch: python and selectors aren't working with **noarch** packages. So the correct usage must be `importlib-metadata` **without** pinning

Reply:
@skupr-anaconda: Without this pinning, building dependent packages (specifically [`swagger-ui-bundle`](https://cloud.prefect.io/anaconda/task-run/53cf09cc-2e6f-49f8-bbc7-4a11d38da4f3?logs=&logId=ab90ce63-c5c8-4b3f-8e41-a3bad6610daf)) fails as it attempts to use `importlib-metadata 4.11.3`.  Is there any way to implement this pinning, or would it just be simpler to update `flake8` to a new version altogether?

Reply:
This can help https://github.com/AnacondaRecipes/swagger-ui-bundle-feedstock/pull/1#issuecomment-1297391356

Reply:
Yes, you can update flake8 to a newer version. But check the flake8's changelog if there are broken changes or no. Because flake8 often is used by CI systems and they can fail